js/lib/beautify.js

```diff
@@ -3,7 +3,7 @@
 
   The MIT License (MIT)
 
-  Copyright (c) 2007-2013 Einar Lielmanis and contributors.
+  Copyright (c) 2007-2015 Einar Lielmanis and contributors.
```

Comment:
If you must do this put it in another commit and do it everywhere.  It is unrelated to this commit.